src/docs/product/data-management-settings/event-grouping/fingerprint-rules.mdx

```diff
@@ -15,7 +15,9 @@ These rules can be configured on a per-project basis in **[Project] > Settings >
 1. Identify the match logic for grouping issues together.
 1. Set the match logic and the fingerprint for it.
 
-The syntax follows the syntax from [Discover queries](/product/discover-queries/). If you want to negate the match, prefix the expression with an exclamation mark (`!`).
+The syntax follows the [syntax](/product/sentry-basics/search/#syntax) from **Discover** queries. If you want to negate the match, prefix the expression with an exclamation mark (`!`).
+
+If you want to create a rule that includes square brackets `[ ]`, replace them with asterisks `*`.
```

Comment:
Was there an issue with square brackets that we or a customer ran into?

Reply:
Yep, see issue [5723](https://github.com/getsentry/sentry-docs/issues/5723)

Reply:
Thanks, sorry I somehow missed that linked issue. I will investigate why the square brackets don't work, maybe we can actually fix this in the code.

Reply:
No worries! I'll leave this open and wait for you to circle back.

Reply:
Turns out that brackets need to be escaped because they are part of [glob syntax ](https://en.wikipedia.org/wiki/Glob_(programming)#Syntax). Globbing is already mentioned under `## Matchers`, so instead of adding this line here, I would expand the explanation there to

```markdown
## Matchers

Matchers allow you to use [glob patterns](https://en.wikipedia.org/wiki/Glob_(programming)).
If you want to create a rule that includes square brackets `[ ]` or a literal `*`, escape them with a backslash, that is, `\[`, `\]` or `\*`. 

The following matchers are available:
```

Reply:
Makes sense to me!